packages/grub2/config/grub.cfg

```diff
@@ -1,35 +1,25 @@
 set timeout=10
 
-# Load custom env file
 set env_file="/grubenv"
-search --no-floppy --file --set=env_blk "${env_file}"
-
-# Load custom env file
 set oem_env_file="/grub_oem_env"
-search --no-floppy --file --set=oem_blk "${oem_env_file}"
-
-# Load custom config file
-set custom_menu="/grubmenu"
-search --no-floppy --file --set=menu_blk "${custom_menu}"
-
-# Load custom config file
-set custom="/grubcustom"
-search --no-floppy --file --set=custom_blk "${custom}"
+set custom_file="/grubcustom"
 
-if [ "${oem_blk}" ] ; then
-  load_env -f "(${oem_blk})${oem_env_file}"
+if [ -f "${oem_env_file}" ]; then
+  load_env -f "${oem_env_file}"
 fi
 
-if [ "${env_blk}" ] ; then
-  load_env -f "(${env_blk})${env_file}"
+search --no-floppy --set oem_blk --label "${oem_label}"
+
+if [ -f "(${oem_blk})${env_file}" ]; then
+  load_env -f "(${oem_blk})${env_file}"
```

Comment:
I'd load fist the `env_file` and after the `oem_env_file`, so the `oem_env_file` has preference in case of a variable overlap.

Reply:
Hmm, isn't the `oem_label` set in `oem_env_file`?

Reply:
mm good point 😅